src/view/layout/_header.njk

```diff
@@ -12,6 +12,10 @@
           </a>
       </div>
 
+      <span style="color:red;font-size:75%;">
```

Comment:
Since you asked for more comments on non-trivial code:
I would argue that could somehow make clear what this is for. If someone would look at this on the next upstream merge conflict, he could not know what to do with that and remove it. 
Since the variable should be empty for builds not by _CircleCI_, so I'm fine if we do not add more conditional rendering to remove it completely from the DOM. But you could consider that to keep it clean. Maybe such condition (like only for development) could already replace the comment. Knowing that it is for development and it is called `WHOAMI`... :man_shrugging:

Reply:
I'll add some later...

Reply:
I think it would be good to exclude that in a production build. Currently if I see it correctly if I would have accidentally set WHOAMI locally and then run `yarn deploy` this would end up on the website. 
Should be easy to do with nunjucks.